bulk.go

```diff
@@ -294,6 +306,9 @@ func (b *Bulk) Run() (*BulkResult, error) {
 				break
 			}
 		}
+		action.idxs = action.idxs[0:0]
+		action.docs = action.docs[0:0]
+		actionPool.Put(action)
```

Comment:
This needs to be above the `if !ok` check - if an operation fails the actions leak.